magento/merchdocs#: adds item to the table in password options for customer configuration section (#1082)
magento/merchdocs: src/configuration/customers/customer-configuration.md

|Reset Password Template|Store View|Determines the email template that is used when customers reset their passwords.|
|Password Template Email Sender|Store View|Determines the store contact that appears as the sender of password-related emails.|
|Recovery Link Expiration Period (hours)|Global|Specifies the number of hours before a password recovery link expires.|

> |Enable Autocomplete on login/forgot password forms|Website|Determines if you want to enable or disable the autocomplete on login/forgot password forms. Options: Yes / No|

Reviewer: ```suggestion
|Enable Autocomplete on login/forgot password forms|Website|Determines if autocomplete is enabled on login/forgot password forms. Options: Yes / No|
```
